pegasus/sites.v3/code.org/public/hourofcode2022.haml

```diff
@@ -0,0 +1,103 @@
+---
+title: Explore, Play, and Create with Computer Science
+theme: responsive
+---
+
+%link{href: "/css/generated/hoc2022-landing-page.css", rel: "stylesheet"}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
```

Comment:
We should avoid taking dependencies on external domains.  Strict firewalls can block them!

Reply:
This should get resolved w/ the work Denys is doing to add FontAwesome Pro to the sites.